database/seed.js

```diff
@@ -1,17 +1,22 @@
 var Movie = require('./index.js').Movie;
-var data = require('./data.js');
+var dataMovie = require('./data.js');
 
-Movie.remove({}, function(err) { 
-  console.log('collection removed'); //empty the movie collection
-});
 
-data.createObject(data.urls, data.thumbnails)
-  .then((results) => {
-    Movie.insertMany(results)
-      .then((results) => {
-        console.log('saved in database');
-      })
-      .catch((err) => {
-        console.log('there was an error saving into the database', err);
-      }); 
+seedDatabase = function(collection, data) {
+  collection.deleteMany({}, function(err) { 
```

Comment:
- On lines 5 & 6 you use a ES5 function instead of ES6, but later in your code use the ES6 arrow functions. Is there a reason for the anonymous ES5 functions rather than ES6? if there isn't a logic issue, I think it would be cleaner to implement the arrow functions throughout this file for consistency across ECMAScript versions.

Reply:
Good catch. I'll change everything to es6 to keep the consistency